doc/nrf/releases/release-notes-1.7.0.rst

```diff
@@ -409,7 +409,7 @@ The following list summarizes the most important changes inherited from upstream
 * Added support for simultaneous multi-image DFU for the nRF53 Series.
   For more information, see the :ref:`ug_nrf5340` and :ref:`ug_thingy53` user guides.
 * Added AES support for image encryption (based on mbedTLS).
-* MCUboot serial: Ported encoding to use the cddl-gen module (which removes dependency on the `TinyCBOR`_ library).
+* MCUboot serial: Ported encoding to use the zcbor module (which removes dependency on the `TinyCBOR`_ library).
```

Comment:
We do not edit the already released release notes unless in critical cases. Is this such a case?
Regardless, I would suggest not making this edit, also because it would suggest ``zcbor`` was used as the name of this module already back in 1.7.0.

Reply:
I agree, will revert.